lib/src/interfaces.dart

```diff
@@ -16,64 +16,63 @@ part of pdfjs;
 
 class AnnotationLayerBuilderOptions {
   JsObject _jsInternal;
-  IL10n _l10n;
-  PDFPageProxy _pdfPage;
+  IL10n? _l10n;
+  PDFPageProxy? _pdfPage;
 
-  AnnotationLayerBuilderOptions() {
+  AnnotationLayerBuilderOptions() :
     _jsInternal = JsObject.jsify({});
-  }
 
-  IL10n get l10n => _l10n;
-  set l10n(IL10n l10n) {
+  IL10n? get l10n => _l10n;
+  set l10n(IL10n? l10n) {
     _jsInternal['l10n'] = l10n;
 
     _l10n = l10n;
   }
 
-  DivElement get pageDiv => _jsInternal['pageDiv'] as DivElement;
-  set pageDiv(DivElement pageDiv) {
+  DivElement? get pageDiv => _jsInternal['pageDiv'] as DivElement?;
+  set pageDiv(DivElement? pageDiv) {
     _jsInternal['pageDiv'] = pageDiv;
   }
 
-  PDFPageProxy get pdfPage => _pdfPage;
-  set pdfPage(PDFPageProxy pdfPage) {
-    _jsInternal['pdfPage'] = pdfPage._jsInternal;
+  PDFPageProxy? get pdfPage => _pdfPage;
```

Comment:
Used by w_viewer to help render individual pages: https://github.com/Workiva/W_viewer/blob/88755a4d51eda6a281f494418a9274b5d4e7d2f3/lib/src/modules/content_renderer/renderers/pdfjs_renderer.dart#L10